lib/process-manager.ts

```diff
@@ -553,9 +553,16 @@ export class QueryProcessManager<T = string, U = string> extends ProcessManager<
 				response => process.send!(`${taskId}\n${JSON.stringify(response)}`)
 			);
 		});
-		process.on('disconnect', () => {
-			process.exit();
-		});
+		process.on('disconnect', () => process.exit());
```

Comment:
ಠ_ಠ

Reply:
b-but it was already there ;_;

Reply:
No, I mean, you changed it for no reason.

The old version doesn't depend on the return value of `process.exit()`, the new version returns it.

Reply:
Ah.